[ Fix ]  위치정보받기 로직 수정

chae-dahee opened the pull request:
## 📌 이슈 및 개요
의논한 위치정보받기 로직으로 수정
병권님이 개발하신 주소검색 창이 안열리는 버그 발생.. 확인 부탁드립니다..!

## 🚀 작업 사항
- 현재 위치 정보받기 의논한 로직으로 수정
- 카카오 주소검색 버그 발생 🐛
1. 마이페이지에 첫 접속
    1. 현재위치정보 버튼
        1. 지도를 이전에 접속해서, 위치정보를 전역으로 가지고 있을 경우 : 로직 3번
    2. 주소검색 버튼
    3. ‘현재 입력된 주소가 없습니다’
2. 현재위치정보 버튼 클릭시
    1. 위치정보동의 알림
3. 동의 이후
    1. 현재위치정보동의 버튼 문자가 ‘다시 현재 위치 받기’ 문자로 변경
    2. ‘위치동의완료’ 아니면 아예 빈문자열 (추후 논의)

## 📸 스크린샷
![image](https://github.com/user-attachments/assets/1a099392-5b9d-47dc-b89b-c56265044fac)

startartart commented on src/pages/MyPage/index.tsx:
```diff
@@ -99,13 +104,7 @@ const MyPage = () => {
+          <p className="text-xl font-bold  text-mainColor w-[25%]">위치 저장</p>
           <LocationBox handleAddress={(address) => addAddress(address)} />
         </div>
         {addresses.length ? (
```
이 부분이 addresses가 있을 때만 호출되는 것으로 보입니다. 한번 확인 부탁드려요

startartart committed:
[ Deploy ] 배포 테스트 (#26)

* setting: eslint, svgr 설정

* feat: 전역 레이아웃 설정

* feat: 헤더, nav바 구성 및 세팅

* [ Feature ] 레이아웃 구성(#2)

* setting: eslint, svgr 설정

* feat: 전역 레이아웃 설정

* feat: 헤더, nav바 구성 및 세팅

* chore: 라이브러리 추가 및 tailwind custom 추가

* style: 스크롤 및 input number 제거

* feat: Button 컴포넌트 구현

* feat: filter-modal 추가

* feat: 컴포넌트 구현
FilterBox, Modal, Overview, SearchBar

* style: Layout Content 수정

* feat: 가게 데이터 레이아웃 구성

* style: theme 추가 및 수정

* style: Modal style 수정

* style: 기타 코드 포매팅

* fix: eslint 에러

* feat: 카카오맵 세팅

* fix: css, 주석제거

* feat: 카카오맵, 현위치

* [ Feature ] 카카오맵 (#8)

* setting: eslint, svgr 설정

* feat: 전역 레이아웃 설정

* feat: 헤더, nav바 구성 및 세팅

* fix: eslint 에러

* feat: 카카오맵 세팅

* fix: css, 주석제거

* feat: 카카오맵, 현위치

* [ Feature ] 마이페이지 및 로그인 화면 구성 (#7)

* style: Button 컴포넌트 스타일 수정

* style: FilterBox 주석 삭제

* style: Layout 스타일 수정

* feat: StoreOverview 컴포넌트 스타일 수정 및 Props 추가

* feat: KakaoAuth API 구현

* feat: LoginBox 컴포넌트 구현

* feat: 로그인 및 마이페이지 구성, 타입 추가, 라우터 반영

* feat: map customOverlay

* fix: merge main

* feat: geolocation 훅, 유틸, 검증

* feat: 마커 애니메이션

* feat: 가게정보 가로 슬라이드 모달

* feat: 가게 데이터 핀, 정보표시

* [ Feature ] 메인메뉴, 마이페이지 필터링 구현 (#15)

* style: tailwind 스타일 추가

* chore: 시도구 데이터 추가

* style: Button 컴포넌트 스타일 수정

* style: 더미데이터 파일 변경

* feat: CityFilter CheckBox 컴포넌트 구현

* fix: router 코드에러 수정

* feat: 필터링 구현 및 모달 리셋 버튼 추가
전역상태관리 X

* chore: react-daum-postcode 라이브러리
더미데이터 추가
address 타입 추가

* feat: FilterBox 모달 닫기 이벤트 추가

* feat: 위치 저장 찾기 구현

* feat: 찜 필터링 기능 구현

* feat: Address 이벤트 연결

* feat: 모달 이벤트 연결

* style: ModalLayout 스타일 수정
Home Modal open시 blur 및 opacity 조정
모바일 환경 및 모달 사이즈 수정

* [ Feature ] User, Shop 전역상태 관리 구현 및 가게 API 연결 (#17)

* chore: tailwind style 추가

* feat: use, shop 전역 상태관리 store 구현

* feat: FilterBox, CityFilter 컴포넌트 전역 상태 관리
필터 검색 API 구현

* feat: ModalLayout 가게 상태 초기화 이벤트 및 Loading 컴포넌트 구현

* feat: API 필터 정제 함수 구현

* feat: StoreOverview 컴포넌트 전역 상태 관리
페이지네이션 검색 API 구현
가게 상세 페이지 이동 이벤트 연결
이외 컴포넌트 스타일 수정

* feat: SearchBar 컴포넌트 엔터키 이벤트 추가

* feat: Home 컴포넌트 전역 상태 관리
입력 검색 API 구현
가게 데이터 리셋 기능 구현

* feat: Shop 컴포넌트 API 연결

* [ Feature ] 가게 상세 페이지 구현 (#19)

* fix: 모달 디자인

* sytle: 여백, 로그인불가 처리

* feat: 위치동의 수정

---------

Co-authored-by: dah <cdh010126r@gmail.com>
Co-authored-by: chae-dahee <96566750+chae-dahee@users.noreply.github.com>

chae-dahee committed:
Merge branch 'dev' into main
